src/main/java/com/hillel/items_exchange/model/User.java

```diff
@@ -59,4 +59,14 @@ public void addUser() {
         phones.forEach(phone -> phone.setUser(this));
         children.forEach(child -> child.setUser(this));
     }
+
+    public void addChild(Child child) {
+        children.add(child);
+        child.setUser(this);
+    }
+
+    public void addPhone(Phone phone) {
+        phones.add(phone);
+        phone.setUser(this);
+    }
```

Comment:
In my humble opinion. it's not a good idea to use methods (addPhone, addChild) here

Reply:
I fallowed the Hibernate user guid to synchronize both ends a bidirectional association.
https://docs.jboss.org/hibernate/orm/5.4/userguide/html_single/Hibernate_User_Guide.html#associations-one-to-many-bidirectional